install-rpi.sh

```diff
@@ -0,0 +1,174 @@
+    echo
+    grep '^## ' "$0" | sed 's/^## /  /'
+    echo
+    exit 1
+fi
+
+STEP="${STEP:-}"
+if [[ "$STEP" == "" ]]; then
+    get_state STEP
+fi
+
+if [[ "$STEP" == 1 ]]; then
+    echo "backing up from ${BACKUP_SSH}"
+    ssh "${BACKUP_SSH}" sudo tar -c -f - /etc/mijia-homie > "${BACKUP_SSH}.etc.mijia-homie.tar"
+    tar -t -f "${BACKUP_SSH}.etc.mijia-homie.tar"
+
+    inc_step
+fi
+
+if [[ "$STEP" == 2 ]]; then
+    echo "setting up sdcard at $SDCARD for unattended installs"
+    cat > "$SDCARD/wpa_supplicant.conf" << EOF
+country=$WIFI_COUNTRY
+update_config=1
+ctrl_interface=/var/run/wpa_supplicant
+network={
+ scan_ssid=1
+ ssid="$WIFI_SSID"
+ psk="$WIFI_PSK"
+}
+EOF
+    cat "$SDCARD/wpa_supplicant.conf"
+    touch "$SDCARD/ssh"
+    diskutil eject "$SDCARD"
+
+    inc_step
+fi
+
+if [[ "$STEP" == 3 ]]; then
+    ssh-keygen -R "${BOOTSTRAP_SSH#*@}"
+    echo "please plug your sdcard into your raspberry pi and restart"
+    echo "setting up ${BOOTSTRAP_SSH}. If asked for a password, type 'raspberry'"
+    # FIXME: use ssh-import-id here instead
+    ssh-copy-id -oStrictHostKeyChecking=no "${BOOTSTRAP_SSH}"
+
+    inc_step
+fi
+
+if [[ "$STEP" == 4 ]]; then
+    if ! ssh -oConnectTimeout=5 -oStrictHostKeyChecking=no "${FINAL_SSH}" true ; then
+        echo "changing your raspberry pi's hostname and restarting"
+        ssh-keygen -R "$FINAL_HOSTNAME"
+        echo "$FINAL_HOSTNAME" | ssh "${BOOTSTRAP_SSH}" sudo tee /etc/hostname
+        if ! ssh "${BOOTSTRAP_SSH}" grep "$FINAL_HOSTNAME" /etc/hosts; then
+            echo "127.0.1.1        ${FINAL_HOSTNAME%%.*}" | ssh "${BOOTSTRAP_SSH}" sudo tee -a /etc/hosts
+
+            ssh "${BOOTSTRAP_SSH}" sudo reboot
+
+        fi
+        ssh -oStrictHostKeyChecking=no "${FINAL_SSH}" hostname
+        IP=$(ssh-copy-id -oStrictHostKeyChecking=no "${FINAL_SSH}" ip address show dev wlan0 | grep ' inet ' | sed -e 's:/.*::' -e 's/^.* //')
+        ssh-keygen -R "$IP"
+        ssh-keyscan -H "${FINAL_HOSTNAME}" >> ~/.ssh/known_hosts
+        ssh-keyscan -H "${IP}" >> ~/.ssh/known_hosts
+    fi
+    inc_step
+fi
+
+if [[ "$STEP" == 5 ]]; then
+    ssh "${FINAL_SSH}" hostname 
+    ssh "${FINAL_SSH}" 'curl https://sh.rustup.rs -sSf | sh -s -- -y'
+    inc_step
+fi
+
+if [[ "$STEP" == 6 ]]; then
+    # shellcheck disable=SC2086
+    ssh "${FINAL_SSH}" ssh-import-id $SSH_IMPORT_IDS
+    inc_step
+fi
+
+if [[ "$STEP" == 7 ]]; then
+    cat "${BACKUP_SSH}.etc.mijia-homie.tar" | \
+        ssh "${FINAL_SSH}" sudo tar -x -v -f - -C /
+    inc_step
+fi
+
+if [[ "$STEP" == 8 ]]; then
+    ARCH="$(ssh "${FINAL_SSH}" uname --machine)"
+    VERSION="$(git tag | sed -n s/mijia-homie-//p | tail -n1)"
+    if [[ "$ARCH" != aarch64 ]] ; then
+        echo "TODO: think about the other rpi architectures"
+        exit 1
+    fi
+    
+    ssh "${FINAL_SSH}" curl -sSLf "https://github.com/alsuren/mijia-homie/releases/download/mijia-homie-${VERSION}/mijia-homie_${VERSION}_arm64.deb" -o "mijia-homie_${VERSION}_arm64.deb" 
```

Comment:
You can now get this from a Debian repository again, see https://github.com/alsuren/mijia-homie/blob/master/mijia-homie/README.md.